### PR TITLE
FEATURE: New command to crawl all sites

### PR DESCRIPTION
**Pull request by paavo**
With `crawler:crawlsites` all sites (and all their Childnodes) found in Neos Siterepository are crawled. $urlSchemeAndHost is generated from data found in domainRepository (needed in a multi-site setup). 

Thanks for your review @Sebobo

### CODE REVIEW

File: Classes/Command/CrawlerCommandController.php [hunk 1]
```diff
@@ -69,6 +70,36 @@ class CrawlerCommandController extends CommandController
...
+     * @throws HttpException
+     */
+    public function crawlSitesCommand(): void {
+        $sites = $this->siteRepository->findAll();
+        $this->outputLine('Found %d sites', [count($sites)]);
+
+        foreach ($sites as $site) {
+            $siteNodeName = $site->getNodeName();
+            $domain = $this->domainRepository->findOneBySite($site, true);
+            $urlSchemeAndHost = ($domain->getScheme() || 'http') . '://' . $domain->getHostname() . ($domain->getPort() ? ':' . $domain->getPort() : '');
```
Comment (Sebobo) [inline Classes/Command/CrawlerCommandController.php:96]: Would be good to check if a domain has been found here and skip the crawl with some output if not.

**Review by Sebobo — CHANGES_REQUESTED** [review]
Awesome, thx. Just one adjustment needed I think.

File: Classes/Command/CrawlerCommandController.php (hunk 1 above)
Comment (paavo) [reply Classes/Command/CrawlerCommandController.php:96]: Oh yes, this makes sense!

**Review by Sebobo — APPROVED** [review]

**Comment by paavo** [comment]
Thanks for merging @Sebobo 🙏 
May you release a new version too?

**Comment by Sebobo** [comment]
Trying, but first have to fixe a bug in the feature ;)

**Comment by paavo** [comment]
> Trying, but first have to fixe a bug in the feature ;)

Oh bad. Feel free to re-assign it to me 🙈

**Comment by Sebobo** [comment]
Already done, see https://github.com/Sebobo/Shel.Crawler/releases/tag/2.3.0

**Pull request merged**